configure.ac

```diff
@@ -42,21 +42,13 @@ AC_SUBST(GMP_LDFLAGS)
 AC_SYS_IS_CYGWIN
 AC_SYS_IS_DARWIN
 
-# External libraries configuration
-EXTERN="\$(CURDIR)/bin/$TARGET/extern"
-MAKE_LIBTARGETS=""
-
 LT_LIB_M
 AC_CHECK_MPFR
 AC_CHECK_MPFI
 AC_CHECK_MPC
 AC_CHECK_CXSC
 AC_CHECK_FPLLL
 
-if ! test -z $MAKE_LIBTARGETS; then # disable external compilation
-    AC_ERROR([Compilation of the external libraries $MAKE_LIBTARGETS was requested, but is now disabled. Please install them manually and rerun configure.])
-fi
```

Comment:
@laurentbartholdi OK, so I understand you want to "keep" support for helping the user build versions of the external dependencies. But then I'd like to know how the current `float` build system enables this: my impression was that as soon as the relevant code is triggered, one runs into the above `AC_ERROR`, and so the code this PR removes seemed unusable.

Clearly I am missing something; if I knew what, I could perhaps revise this PR to satisfy both needs (i.e. mine, which is it that `./configure && make` should just work, vs. that of those people who'd like `float` to build external deps for them)